src/components/Switch/Switch.scss

```diff
@@ -0,0 +1,109 @@
+@import "constants/style";
+
+$padding--default: $spacing--xs;
+$padding--mobile: 6px;
+
+.switch {
+  all: unset;
+  user-select: none;
+
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  gap: 0;
```

Comment:
There should be a small gap imo

Reply:
maybe yeah, for now it's in line with the design though.